test/config-luatest/config_test.lua

```diff
@@ -281,3 +281,97 @@ g.test_config_option = function()
     t.assert_equals(res.exit_code, 0)
     t.assert_equals(res.stdout, table.concat({16, 100000000, 0}, "\n"))
 end
+
+g.test_no_cluster_config_failure = function()
```

Comment:
We agreed to try to re-write this test using test.luatest_helpers.server instance to ensure that the instance is operational after failed reload and to better structurize the testing code.

Reply:
Done